crates/typst/src/export/pdf/page.rs

```diff
@@ -296,20 +289,55 @@ impl PageContext<'_, '_> {
 
     fn set_fill(&mut self, fill: &Paint) {
         if self.state.fill.as_ref() != Some(fill) {
-            let f = |c| c as f32 / 255.0;
             let Paint::Solid(color) = fill;
             match color {
-                Color::Luma(c) => {
+                Color::Luma(_) => {
```

Comment:
Perhaps this whole function should be moved into the `color` file, too? Could be a free-standing function that takes `&mut Content`. Or something else if you have a good idea.